### PR TITLE
fix(accident-notification): review screen does not update

### PR DESCRIPTION
**Pull request by sigruntg**
# Review screen does not update

[1nbj5y5](https://app.clickup.com/t/1nbj5y5)

## What

* Added poll interval to getAccidentStatus so the user sees changes every 5 minutes even if he/she does not refresh.
* Changed the way we get error message for received documents on review status screen.
* Added the same check to add attachement template so the user does not see the it if he/she has not refreshed the page.

## Why

There was a problem when adding documents. When you went back you still saw the item in the list because we got this information from a service. I changed it so we also check answers for this information.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by bensijons — APPROVED** [review]
LGMT

File: libs/application/templates/accident-notification/src/forms/InReviewForm/addAttachmentsSection.ts [hunk 1]
```diff
@@ -55,7 +57,8 @@ export const addAttachmentsSection = (isAssignee?: boolean) =>
             condition: (formValue) =>
               isPowerOfAttorney(formValue) &&
               !isUniqueAssignee(formValue, !!isAssignee) &&
-              !hasReceivedProxyDocument(formValue),
+              isReportingOnBehalfOfInjured(formValue) &&
```
Comment (Thorrik58) [inline libs/application/templates/accident-notification/src/forms/InReviewForm/addAttachmentsSection.ts:60]: juridicialperson should not have to upload the proxy doc

File: libs/application/templates/accident-notification/src/utils/isFatalAccident.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { YesOrNo } from '../types'
 export const isFatalAccident = (formValue: FormValue) => {
   const wasTheAccidentFatal = getValueViaPath(
     formValue,
-    'wasTheAccidentFata',
```
Comment (Thorrik58) [inline libs/application/templates/accident-notification/src/utils/isFatalAccident.ts:8]: lol 🪣

File: libs/application/templates/accident-notification/src/forms/InReviewForm/addAttachmentsSection.ts (hunk 1 above)
Comment (Thorrik58) [inline libs/application/templates/accident-notification/src/forms/InReviewForm/addAttachmentsSection.ts:60]:
```suggestion
```

File: libs/application/templates/accident-notification/src/forms/InReviewForm/addAttachmentsSection.ts [hunk 3]
```diff
@@ -64,7 +67,8 @@ export const addAttachmentsSection = (isAssignee?: boolean) =>
...
```
Comment (Thorrik58) [inline libs/application/templates/accident-notification/src/forms/InReviewForm/addAttachmentsSection.ts:70]:
```suggestion
```

File: libs/application/templates/accident-notification/src/forms/InReviewForm/addAttachmentsSection.ts [hunk 4]
```diff
@@ -76,21 +80,19 @@ export const addAttachmentsSection = (isAssignee?: boolean) =>
...
```
Comment (Thorrik58) [inline libs/application/templates/accident-notification/src/forms/InReviewForm/addAttachmentsSection.ts:83]:
```suggestion
```

**Review by Thorrik58 — CHANGES_REQUESTED** [review]
juridicial person should not have to upload the proxy document

**Review by Thorrik58 — APPROVED** [review]

**Pull request merged**